repl/src/main/resources/fake_shell.py

```diff
@@ -368,6 +369,11 @@ def main():
     sys.stderr = fake_stderr
 
     try:
+
+        if os.environ["LIVY_TEST"] == "false":
```

Comment:
Or rather, just store it in a variable:

    is_testing = os.environ.get('LIVY_TEST') == "true"

And use it later when cleaning up also.